### ISSUE TITLE
How can I get attention weight durning TTS trianing?

### ISSUE THREAD

**Issue by iiynjgef** [espnet/espnet#5452]
I trained the tacotron2 model in "egs2/ljspeech". 
And I get the att_w in inference mode (`model.eval()` is used in Text2Speech class) by the following code:

```
text2speech1 = Text2Speech.from_pretrained(
  model_file=tacotron2_trained,
  vocoder_file=vocoder_path,
... ...

 with torch.no_grad():
  results = model(text)
  att_w = results["att_w"]
``` 

But I didn't find any code which returns the "att_w" by `model.train()`.
**Can the attention weights be returned in training mode? How to do?**
